### ISSUE TITLE
Docs: Accessibility options are missing

### ISSUE THREAD

**Issue by pawelfus** [highcharts/highcharts#10079]
Probably the main problem is with missing type for `lang.accessibility`.

List of missing options:

1. `accessibility.series`
2. `accessibility.longDescriptionHeading`
3. `accessibility.rangeSelectorMaxInput`
4. `accessibility.structureHeading`
5. `accessibility.chartContainerLabel`
6. `accessibility.mapZoomOut`
7. `accessibility.chartTypes`
8. `accessibility.axis`
9. `accessibility.rangeSelectorMinInput`
10. `accessibility.rangeSelectorButton`
11. `accessibility.exporting`
12. `accessibility.legendItem`
13. `accessibility.tableSummary`
14. `accessibility.noDescription`
15. `accessibility.viewAsDataTable`
16. `accessibility.defaultChartTitle`
17. `accessibility.screenReaderRegionLabel`
18. `accessibility.mapZoomIn`
19. `accessibility.chartHeading`
20. `accessibility.svgContainerTitle`
21. `accessibility.navigationHint`
22. `accessibility.seriesTypeDescriptions`

Series:
1. `series.yAxisDescription`
2. `series.xAxisDescription`
3. `series.summary`

Exporting:
1. `exporting.exportRegionLabel`
2. `exporting.menuButonLabel`
3. `exporting.chartMenuLabel`
